src/codeu/chat/server/Server.java

```diff
@@ -53,23 +54,31 @@
   private final Timeline timeline = new Timeline();
 
   private final Map<Integer, Command> commands = new HashMap<>();
-
   private final Uuid id;
   private final Secret secret;
-
+  private ServerInfo info = null;
   private final Model model = new Model();
   private final View view = new View(model);
   private final Controller controller;
-
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
+  // This string should be changed whenever the initial version value for the server
+  // needs to be changed
+  private String version = "1.0.0";
```

Comment:
small note: this is no longer the initial version of the server, is it? This is the current version of the server.